stdlib/2/_functools.pyi

```diff
@@ -12,9 +16,72 @@ def reduce(function: Callable[[_T, _T], _T],
 def reduce(function: Callable[[_T, _S], _T],
            sequence: Iterable[_S], initial: _T) -> _T: ...
 
-class partial(object):
-    func = ...  # type: Callable[..., Any]
-    args = ...  # type: Tuple[Any, ...]
-    keywords = ...  # type: Dict[str, Any]
-    def __init__(self, func: Callable[..., Any], *args: Any, **kwargs: Any) -> None: ...
-    def __call__(self, *args: Any, **kwargs: Any) -> Any: ...
+@overload
+def partial(func: Callable[[_T], _S], arg: _T) -> Callable[[], _S]: ...
```

Comment:
Could you mark all arguments as positional only by prefixing them with two underscores, like `__func` and `__arg`?

Reply:
done